src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs

```diff
@@ -62,7 +59,6 @@ public void PopulateValues(ViewLocationExpanderContext context)
             foreach (var location in viewLocations)
             {
                 yield return location.Replace("{0}", value + "/{0}");
-                yield return location;
```

Comment:
One of the reasons this expander uses a `yield return` is because you could chain multiple expanders together and the number of strings that you generate will grow very quickly. In general, the expander should avoid creating strings until they're actually iterated upon.

Additionally appending the original list at the end might not be the correct behavior - for instance we should prefer `Views/Home/_Partial.cshtml` to `Views/Shared/{locale}/_Partial.cshtml`